great_expectations/expectations/metrics/column_map_metrics/column_values_json_parseable.py

```diff
@@ -26,7 +26,7 @@ def is_json(val):
         return column.map(is_json)
 
     @column_condition_partial(engine=SparkDFExecutionEngine)
-    def _spark(cls, column, json_schema, **kwargs):
+    def _spark(cls, column, **kwargs):
```

Comment:
This was the bug. `json_schema` is not required by the `ColumnValuesJsonParseable` metric, but was required by the `_spark` partial function.